08-rails/code_samples/destroy_controller_tests.rb

```diff
@@ -0,0 +1,29 @@
+describe "destroy" do
+  describe "destroy" do
+    it "can destroy a model" do
+      # Arrange
+      poodr = Book.new title: "Practical Object Oriented Programming in Ruby", author: "Sandi Metz"
```

Comment:
This is a little hard to read without parens.

```suggestion
      poodr = Book.new(title: "Practical Object Oriented Programming in Ruby", author: "Sandi Metz")
```